conda/recipes/libcudf/recipe.yaml

```diff
@@ -244,34 +220,23 @@ outputs:
         overlinking_behavior: "error"
     requirements:
       build:
-        - ${{ compiler("c") }}
         - ${{ compiler("cuda") }}
-        - ${{ compiler("cxx") }}
-        - ${{ stdlib("c") }}
```

Comment:
Am curious why the other compiler packages are dropped here? Also why is only `${{ compiler("cuda") }}` kept?

Reply:
Just for future context -- my initial thought was that we could ignore all the compiler exports.  That is incorrect.  Then I thought we could at least ignore the cuda compiler exports, since the version pinning is handled by `cuda-version`, but there are recent changes to account for card variants so we want to keep the cuda compiler exports around, too.